### ISSUE TITLE
documentation inconsistancies

### ISSUE THREAD

**Issue by gapster** [mamba-org/mamba#2907]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### How did you install Mamba?

Mambaforge

### Search tried in issue tracker

mambaforge miniforge

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

Not applicable

### Describe your issue

I was about to write a post about confusion with mamba channels.  In particular, what channel is my mamba actually using.  (I actually wrote the whole post before checking *troubleshooting*).  (Before directing me to [post #656](https://github.com/mamba-org/mamba/issues/656) I've looked at that.  There's too much going on there to extract an answer.)

While drafting my original post I was instructed to check [Troubleshooting](https://mamba.readthedocs.io/en/latest/user_guide/troubleshooting.html) where I found several items concerning channels, and instructions on commands to execute after installing mamba.   It seems to me that those instructions should be part of the *installation instructions*, not *Troubleshooting*.     I don't know if the instructions there will solve my problem, but they might, and if they were part of the installation instructions I wouldn't have spent *hours* trying to understand what channel mamba was actually using, and why there are two .condarc files.  (That is still puzzling me.)

While I'm at it:

The [GitHub site for conda-forge](https://github.com/conda-forge/miniforge) says.
"As of September 2023, the new usage of Mambaforge is thus discouraged. Bug reports specific to Mambaforge will be closed as won't fix.  That said, if you had to start using one today, we recommend to stick to Miniforge."

But [*readthedocs*](https://mamba.readthedocs.io/en/latest/mamba-installation.html) says "
We recommend that you start with the [Mambaforge distribution](https://github.com/conda-forge/miniforge#mambaforge)."

Minor.   *readthedocs* is a little out of date.    (It is, isn't it?)  Is it a fact that there is no difference between miniforge and mambaforge other than the installation location?

### mamba info / micromamba info

```shell
conda version : 23.3.1
    conda-build version : not installed
         python version : 3.10.12.final.0
       virtual packages : __archspec=1=x86_64
                          __win=0=0
       base environment : C:\Users\garyp\miniforge3  (writable)
      conda av data dir : C:\Users\garyp\miniforge3\etc\conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
                          https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : C:\Users\garyp\miniforge3\pkgs
                          C:\Users\garyp\.conda\pkgs
                          C:\Users\garyp\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\garyp\miniforge3\envs
                          C:\Users\garyp\.conda\envs
                          C:\Users\garyp\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/23.3.1 requests/2.31.0 CPython/3.10.12 Windows/10 Windows/10.0.19045
          administrator : False
             netrc file : None
           offline mode : False
```

### ~/.condarc

```yaml
(base) C:\Users\garyp>type   C:\Users\garyp\miniforge3\.condarc && type CON
channels:
- conda-forge


(base) C:\Users\garyp>type   C:\Users\garyp\.condarc && type CON
channels:
  - defaults
ssl_verify: true
channel_priority: strict
```

**Comment by jonashaag** [comment]
Thanks for the feedback! We should update the docs to reflect that Mambaforge is now an alias to Miniforge.

Re Troubleshooting vs installation guide, I'm not sure the channel business should be part of the installation guide. If you start from a fresh environment with no previous installation of (Ana)Conda then you end up with a consistent, working setup:

```
$ curl -L -O "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
$ bash Miniforge3-$(uname)-$(uname -m).sh
... say "yes" to everything ...
$ mamba info
...
           channel URLs : https://conda.anaconda.org/conda-forge/linux-aarch64
                          https://conda.anaconda.org/conda-forge/noarch
...
```

What installation instructions did you follow? I'm not sure how you ended up with the Anaconda channels in your environment but it's not part of the installation instructions.

Maybe we should add to the installation instructions to check that you don't have the Anaconda channels.

**Comment by jonashaag** [comment]
Could it be that `C:\Users\garyp\.condarc` was created by some older Conda install from sometime in the past? In any case I think you should just delete it

**Comment by gapster** [comment]
Yep.   Deleted.   I guess that sort-of answers my questions about multiple .condarc files.   Did I miss an instruction to delete the old one if I have one?  I'm going to delete that environment and install everything in a new one.   The packages in the current env probably came from default, except for jupyter-lab

```
(test) C:\Users\garyp>conda config --show channels
channels:
  - conda-forge

(test) C:\Users\garyp>conda config --show-sources
==> C:\Users\garyp\miniforge3\.condarc <==
channels:
  - conda-forge
```
